mozilla_schema_generator/probes.py

```diff
@@ -51,39 +51,8 @@ class MainProbe(Probe):
 
     first_added_key = "first_added"
 
-    # TODO: Do we use log_sum_*, sum_squares_*?
     histogram_schema = {
-        "type": "object",
-        "properties": {
-            "bucket_count": {
-                "minimum": 0,
-                "type": "integer"
-            },
-            "histogram_type": {
-                "minimum": 0,
-                "type": "integer"
-            },
-            "range": {
-                "items": {
-                    "type": "integer"
-                },
-                "type": "array"
-            },
-            "sum": {
-                "minimum": 0,
-                "type": "integer"
-            },
-            "values": {
-                "additionalProperties": False,
-                "patternProperties": {
-                    "^[0-9]+$": {
-                        "minimum": 0,
-                        "type": "integer"
-                    }
-                },
-                "type": "object"
-            }
-        }
+        "type": "string",
```

Comment:
There is one failing test that I haven't been able to modify successfully, as I'm not yet familiar enough with the plumbing here. In `test_integration.py`, `test_max_size` fails to split. I tried reducing max_size down to 6, but it still doesn't split. If I reduce further, I get an error about max_size being smaller than env. Should I add additional probes in order to increase the column count on this test?